### DISCUSSION TITLE
Samba Not Starting

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1082] [Fixed Issues]
@rern 

I have to restart the smb service after a boot, although it is enabled in the menu.

**Comment by R101G** [comment]
@rern 

Just to add that _systemctl status smb_  shows that samba is running, but connection is refused until I run _systemctl restart smb_ 

The status output does not change.

**Comment by R101G** [comment]
This looks like a common problem where samba starts before the network port is up. Changing smb.conf entry to bind interfaces only = no, rather than yes, fixes it.

EDIT: Another way which seems to work and  does not generate error messages, is simply to enable nmb. It seems nmb waits for the network interface, and smb waits for nmb.

**Comment by rellimneb** [comment]
hi i have the same issue, i preety new to linux stuff so i don't quite understand how to implement your fix.  How do you enable nmb?  thanks

**Comment by R101G** [comment]
You have to login via ssh, then I think it was just `systemctl enable nmb` and then `systemctl start nmb`

If you have not used ssh before, go to the terminal utility and enter `ssh root@raudio.local `and the password `ros `

**Reply by rellimneb** [reply]
perfect, thanks! that did the trick

**Comment by rern** [comment]
`bind interfaces only` was removed, default as `no`, by 20221117 update as of @R101G suggestion.
